drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1596,9 +1596,13 @@ void TextureStorage::_update_render_target(RenderTarget *rt) {
 			glTexParameteri(texture_target, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
 			glTexParameteri(texture_target, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
 		}
+#ifndef IOS_ENABLED
 		if (use_multiview) {
 			glFramebufferTextureMultiviewOVR(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, rt->color, 0, 0, rt->view_count);
```

Comment:
@dsnopek do you know if it is possible to support multiview on iOS devices?

Reply:
Unfortunately, I don't know for sure, but I strongly suspect that multiview is not supported on iOS.

Reply:
AFAIK, it's not supported on any iOS device.